reporter/src/funTest/assets/copyright-statements-expected-output.yml

```diff
@@ -1391,6 +1391,10 @@ processed_statements:
   - "(c) 2008 thawte, Inc."
   Copyright 2014 the Melange authors.:
   - "Copyright 2014 the Melange authors."
+  copyright 2000-2005 u' CDE:
```

Comment:
Commit message nit: "test case" should probably be "test data", because I was looking for an actual test case implementation added in Kotlin code in this commit.